### PR TITLE
feat(infra): collect more data to indexer

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#8549** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8549?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8528** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8528?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class DocsIndexer extends Entity {
   /**
    * increase this number to re-index all docs
    */
-  static INDEXER_VERSION = 2;
```
Comment (pengx17) [inline packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts:39]: Gabe, is that you?

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts (hunk 1 above)
Comment (EYHN) [reply packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts:39]: ?

Comment (pengx17) [reply packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts:39]: I saw 3 is skipped.

File: packages/common/infra/src/utils/__tests__/match-url.spec.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	http://foo.com/(something)?after=parens
+	http://foo.com/blah_blah.
+	http://foo.com/blah_blah/.
+    6:00p
+    filename.txt
+    www.example.com
+    WWW.EXAMPLE.COM
+    “is.gd/foo/”
+    http://example.com/something?with,commas,in,url, but not at end
+	What about <mailto:gruber@daringfireball.net?subject=TEST> (including brokets).
```
Comment (forehalo) [inline packages/common/infra/src/utils/__tests__/match-url.spec.ts]: 。“，【】「」《》

**Pull request merged**